components/tags/SubjectTag.tsx

```diff
@@ -25,6 +26,19 @@ export const SubjectTag = ({
       )
     }
   }
-  
+
+  if (tag.startsWith('report:')) {
```

Comment:
Unrelated to the PR but workspace items are super cramped for space nowadays due to multiple tags per item with report type. The icon display saves us some real estate while not losing any meaning.